app/models/trip.rb

```diff
@@ -0,0 +1,13 @@
+class Trip < ApplicationRecord
+  belongs_to :driver
+  belongs_to :passenger
+  validates :cost, numericality: true
+
+  def cost_in_dollar
+    sprintf('%.2f', (self.cost / 100))
+  end
+
+  def pretty_date
```

Comment:
👍 

You'll learn later about view_helpers and that might be more useful later as a view helper.